packages/components/src/tabs/styles.ts

```diff
@@ -234,6 +234,9 @@ export const Tab = styled( Ariakit.Tab )`
 
 export const TabChildren = styled.span`
 	flex-grow: 1;
+
+	display: flex;
+	align-items: center;
```

Comment:
It seems that tabs with icons that have custom wider widths will have icons that aren't centered:

![Screenshot 2024-10-18 at 11 45 42](https://github.com/user-attachments/assets/9f847a3b-bcc5-4be1-93a7-23c02ebc781c)

Do we also need a `justify-content: center`:


```suggestion
	align-items: center;
	justify-content: center;
```